src/Algebra/Definitions.agda

```diff
@@ -161,3 +161,15 @@ LeftDivides ∙ \\ = (LeftDividesˡ ∙ \\) × (LeftDividesʳ ∙ \\)
 
 RightDivides : Op₂ A → Op₂ A → Set _
 RightDivides ∙ // = (RightDividesˡ ∙ //) × (RightDividesʳ ∙ //)
+
+StarRightExpansion : A → Op₂ A → Op₂ A → Op₁ A → Set _
+StarRightExpansion e _+_ _∙_ _⁻* = ∀ x → (e + (x ∙ (x ⁻*))) ≈ (x ⁻*)
```

Comment:
Hmm these names don't quite match thematically with those already in the file. Every other name is either an adjective or adjectival. If possible I'd prefer to stick to that style. Do you have any ideas?

Reply:
`StarRightExpansive`?

Reply:
Sounds good what about for `fixedPoint`?

Reply:
Hmm actually looking at the definitions of the `FixedPoint` names, I'm a bit worried about the name `FixedPoint` altogether. It's not a "standard" fixed point, e.g. `f x = x`. Is there a more precise name from the literature that you know of @Akshobhya1234 ?

Reply:
I have seen `StarDestructLeft` and `LeftInduction` in some literature. So it can become `StarDestruct` and `Induction` when we combine left and right.

Reply:
Sounds good to me!